.build-test-rules.yml

```
@@ -28,3 +28,8 @@ components/esp_lvgl_port/examples/rgb_lcd:
   disable:
     - if: IDF_VERSION_MAJOR < 5
       reason: Example for RGB LCD is supported only for IDF >= 5.0
+
+components/lcd/esp_lcd_ili9881c:
+  disable:
+    - if: (IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR < 3) or IDF_VERSION_MAJOR < 5 or IDF_TARGET != "esp32p4"
```

Please, could you try this 
```suggestion
    - if: ((IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR < 3) or IDF_VERSION_MAJOR < 5) and IDF_TARGET != "esp32p4"
```

Or you can try this: 
```
- if: ((IDF_VERSION_MAJOR == 5 and IDF_VERSION_MINOR < 3) or IDF_VERSION_MAJOR < 5) and IDF_TARGET not in ["esp32p4"]
```

---

Alright, perhaps it's a mistake in the syntax I used. I will make the necessary adjustments and try again.

---

I tried on my side and fixed more issues here: https://github.com/espressif/esp-bsp/pull/373
1. File `.build-test-rules.yml`
2. Test app in esp_lcd_7796 Cmake changed and YML too (there is function, which is not supported in IDF 4.4 (I added condition for IDF 5.0 and higher)

Now, it should pass CI.

---

Thank you very much for your help! I will sync this modification and submit it again.

---

You are welcome :-) Please, did you signed CLA (Contributor License Agreement) ?  It seems, that CI is still waiting for it.

---

Sorry, there were some issues with the CLA earlier, but they have been fixed now. You can merge the PR.